packages/foreman/nodejs-react-html-parser/nodejs-react-html-parser.spec

```diff
@@ -0,0 +1,52 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name react-html-parser
+
+Name: %{?scl_prefix}nodejs-react-html-parser
+Version: 2.0.2
+Release: 1%{?dist}
+Summary: Parse HTML into React components
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/wrakky/react-html-parser#readme
+Source0: https://registry.npmjs.org/react-html-parser/-/react-html-parser-%{version}.tgz
+%if 0%{?!scl:1}
+BuildRequires: nodejs-packaging
+%endif
+Requires: npm(htmlparser2) >= 3.9.0
```

Comment:
Looks like this should also be looking at the SCL package. Not sure why it isn't. You may also need to package that.